src/components/Card.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+	<element :is="isLink ? 'a' : 'div'"
+		:href="href"
+		class="card"
+		:class="{'card--link': isLink }">
+		<div v-if="!isLink" class="card__icon">
+			<slot />
+		</div>
+		<div class="card__text">
+			<h3 class="card__heading">
+				{{ title }}
```

Comment:
not sure but we might need to add aria-labels here?

Reply:
To what part?

Reply:
already discussed with Marco that we apparently dont need it here